### PR TITLE
build: add aarch64-linux-gnu binary to releases

### PR DESCRIPTION
**Pull request by morenol**
Attempt to help with https://github.com/rustwasm/wasm-bindgen/issues/2550

### CODE REVIEW

**Comment by alexcrichton** [comment]
Could you update the diff to not update most of the CI configuration? Ideally this would just be adding a small new section.

**Comment by morenol** [comment]
> Could you update the diff to not update most of the CI configuration? Ideally this would just be adding a small new section.

Sure! the other changes were done by the code editor. It is updated now

**Review by alexcrichton — COMMENTED** [review]
Thanks! Can you confirm that the artifacts at the [bottom of this page](https://github.com/rustwasm/wasm-bindgen/actions/runs/3274843709) work for you?

File: .cargo/config [hunk 1]
```diff
@@ -2,3 +2,6 @@
 # that.
 [target.wasm32-unknown-unknown]
 runner = 'cargo run -p wasm-bindgen-cli --bin wasm-bindgen-test-runner --'
+
+[target.aarch64-unknown-linux-gnu]
+linker = "aarch64-linux-gnu-gcc"
```
[the remark this answers is not in the document]
Comment (morenol) [reply .cargo/config]: Yes, it works for me!

**Pull request merged**